### ISSUE TITLE
Change the number of anchor boxes

### ISSUE THREAD

**Issue by alextudx** [pythonlessons/TensorFlow-2.x-YOLOv3#97]
Dear Madam or Sir,

depending on my understanding, when we reduce the number of anchor boxes, we could improve the FPS of yolo. But when I change the anchor boxes (For example: from 6 to 5 or 6 to 3 by yolov3 tiny, we failed to train the model, the loss is always nan.) Could you tell me why or give me some suggestions to solve this problem?

Best regards
Alex

**Comment by pythonlessons** [comment]
Hi, thanks for the question. But I must say, that I havent tried to change anchor boxes count while writing code for this, so I really can't answer this question to you right now.
